aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js

```diff
@@ -144,9 +144,27 @@ export const TwoColumnWizardExample = () => {
   const [open, setOpen] = useState(false);
   const size = useContext(ResponsiveContext);
 
+  const wizardRef = React.useRef();
```

Comment:
This is great! I think the comments make sense as well so the users know they will not need this